lametro/bills.py

```diff
@@ -121,6 +121,12 @@ def scrape(self, window=28, matter_ids=None) :
 
         n_days_ago = datetime.datetime.utcnow() - datetime.timedelta(float(window))
         for matter in matters:
+            # If this Boolean field is True, then do not scrape the Bill.
+            # This issue explains why a restricted Bill might appear (unwelcome) in the Legistar API:
+            # https://github.com/datamade/la-metro-councilmatic/issues/345#issuecomment-421184826 
+            if matter['MatterRestrictViewViaWeb']:
```

Comment:
is this always true or false and not none?

Reply:
The Legistar docs describe it as a `boolean`: http://webapi.legistar.com/Help/Api/PUT-v1-Client-Matters-MatterId A boolean, I believe, can be none, but in this case, the value of  `MatterRestrictViewViaWeb` is determined Metro staff checking a box. That is, if the box is checked, then the value is `true`, and if the box is unchecked, then the value is `false`.